### DISCUSSION TITLE
About drivers delivration

### DISCUSSION THREAD

**Discussion by Sid8899** [microsoft/wslg#271] [General]
Hi I'm really excited about your work here !

I was asking my self, do you think OEM laptops will have pretty early (let say 1 months) GPU and integrated GPU updates to fully support WSLg ??

Regards

**Comment by spronovo** [comment]
Hi @Sid281216, thanks for the kind words.

All of our graphics partners have WSL/WSLg capable driver for their more recent GPU (see our partner beta drivers for details on how many generation of GPU each of them support). 

In term of specific OEM laptop receiving driver update with this support, that is up to the OEM to chose when/if to pickup and post new driver for their platform. Even if an OEM doesn't update the driver on their platform, often you'll find a web driver from your graphics vendor will install and give you the functionality you are looking for. The decision on which system to support this way it up to the OEM or graphics vendor and not something Microsoft enforces.

**Reply by Sid8899** [reply]
Since WSLg is a huge new capability for windows, they have all interest to apply changes for drivers. Let's wait and see ;)
